public/app/features/search/service/types.ts

```diff
@@ -78,6 +80,13 @@ export interface GrafanaSearcher {
   getSortOptions: () => Promise<SelectableValue[]>;
   sortPlaceholder?: string;
 
+  getFolderChildren: (folderUid?: string) => Promise<DashboardViewItem[]>;
```

Comment:
does this mean it should get the items in the folder, or everything below?

Do we need to worry about paging?

Reply:
It returns immediate children, not all descendants

Reply:
Current folder API does not paginate, but that is required.